src/main/java/frc/robot/subsystems/AlgaeIntake.java

```diff
@@ -13,23 +14,38 @@
 
 public class AlgaeIntake extends SubsystemBase {
   TalonFX intakeMotor;
-  CANrange algaeSensor;
+  TalonFXConfiguration intakeConfig;
+  double intakeHasGamePieceVelocity = constAlgaeIntake.ALGAE_INTAKE_HAS_GP_CURRENT;
+  double intakeHasGamePieceCurrent = constAlgaeIntake.ALGAE_INTAKE_HAS_GP_VEOLOCITY;
 
   /** Creates a new AlgaeIntake. */
   public AlgaeIntake() {
     intakeMotor = new TalonFX(mapAlgaeIntake.ALGAE_MOTOR_CAN);
+    intakeConfig = new TalonFXConfiguration();
+    intakeMotor.getConfigurator().apply(intakeConfig);
```

Comment:
A TalonFX configuration already exists in constants
```suggestion
    intakeMotor.getConfigurator().apply(constAlgaeIntake.ALGAE_INTAKE_CONFIG);
```